apps/molgenis-components/src/components/forms/EditModal.vue

```diff
@@ -204,6 +205,20 @@ export default {
         };
       });
     },
+    saveDraftDisabledMessage() {
+      if (
+        this.tableMetaData?.columns.some(
+          (c) =>
+            c.key == 1 &&
+            c.columnType != "AUTO_ID" &&
+            this.rowData[c.id] == null
+        )
+      ) {
+        return "Cannot save draft: primary key is required";
+      } else {
+        return null;
+      }
```

Comment:
It think it's better for these kind of variables to have '' (empty string) as the 'no error'-value, since empty string is also falsy and it makes the typing way less complicated.